docs/04.guides/04.cookbooks/29.Threads-Scope/page.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Threads
+id: thread_usage
+---
+## Thread Scope ##
+
+This document explains how to use the threads. Threads mainly used for execute the code parallel.
+
+### Example 1 ###
+
+This below example shows waiting normal execution waiting for all code to resolve. Here it waits for 1000 mili seconds after the it complete the execution.
```

Comment:
***ToChange***
The below example shows normal execution, waiting for all code to resolve. Here it takes 1000 milliseconds to complete the execution.